### PR TITLE
Display Let's Encrypt certificate status

### LINKED ISSUE #409: [domains] Let's Encrypt status error is not correctly handled
A domain on the web dashboard:

Screenshot from 2019-04-16 14-04-31

And the same on the CLI:

```
+----------------+--------------------------------+
|     DOMAIN     |              SSL               |
+----------------+--------------------------------+
| *.example.com   | Valid until 2019-04-12         |
|                | 19:40:07 +0200 CEST            |

```

### PR DESCRIPTION
**Pull request by EtienneM**
Fix #409

```
$ go build -o cli ./scalingo && ./cli -a my-app domains
+----------------+--------------------------------+
|     DOMAIN     |            TLS/SSL             |
+----------------+--------------------------------+
| example.com    | (LE In use) Valid until        |
|                | 2019-07-01 10:57:26 +0200 CEST |
| *.example.net  | (LE Error) Valid until         |
|                | 2019-04-12 19:40:07 +0200 CEST |
| *.example.eu   | (LE In use) Valid until        |
|                | 2019-06-12 09:00:11 +0200 CEST |
| example.net    |                              - |
| example.eu     |                              - |
+----------------+--------------------------------+

```

### CODE REVIEW

File: domains/list.go [hunk 1]
```diff
@@ -17,7 +26,7 @@ func List(app string) error {
 	}
 
 	t := tablewriter.NewWriter(os.Stdout)
-	t.SetHeader([]string{"Domain", "SSL"})
+	t.SetHeader([]string{"Domain", "TLS/SSL"})
```
Comment (johnsudaar) [inline domains/list.go]: You could not resist? :stuck_out_tongue:

Comment (EtienneM) [reply domains/list.go]: :laughing:

**Comment by EtienneM** [comment]
I don't like it either, but I'm not sure a new column is better. How do you se this new column? Naming it `Let's Encrypt` and only adding it if there is at least a domain with a Let's Encrypt  certificate?

**Comment by johnsudaar** [comment]
> I don't like it either, but I'm not sure a new column is better. How do you se this new column? Naming it `Let's Encrypt` and only adding it if there is at least a domain with a Let's Encrypt certificate?

Why not yep.

**Comment by yannski** [comment]
Maybe the column should be named "TLS". Therefore Let's Encrypt status and Custom certificate status should be merged? BTW we have the same semantic "problem" on the web dashboard.

**Comment by EtienneM** [comment]
```
$ go build -o cli ./scalingo && ./cli -a my-app domains
+----------------+-----------------------+
|     DOMAIN     |        TLS/SSL        |
+----------------+-----------------------+
| abcde.azert.me | Let's Encrypt: In use |
| *.azerty.net   | Let's Encrypt: Error  |
| *.azerty.eu    | Let's Encrypt: In use |
| azerty.net     |                     - |
| azerty.eu      |                     - |
+----------------+-----------------------+

```

**Comment by johnsudaar** [comment]
And for custom certificates ?

**Comment by EtienneM** [comment]
The code didn't change for custom certificate so it displays `Valid until  2019-07-01 10:57:26 +0200 CEST`

**Pull request merged**